### PR TITLE
Adds default errors for applications

### PR DESCRIPTION
**Pull request by waldemarnt**
Before on equipment api topic:
screen shot 2016-08-19 at 1 43 15 pm
After:
screen shot 2016-08-19 at 1 43 29 pm

An errors are was added to telemetry api as well:
screen shot 2016-08-19 at 1 44 48 pm

### CODE REVIEW

File: source/includes/telemetry_api/_errors.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+## Errors
```
Comment (lucaslago) [inline source/includes/telemetry_api/_errors.md]: What do you think about changing this to "Error Codes"?

File: source/includes/equipment_api/_equipment.md [hunk 2]
```diff
@@ -126,43 +126,41 @@ Refer to the [CAN Variables](#can-variables) section to see the possible returne
...
-
-<blockquote class='lang-specific schema'><p>HTTP 401 Unauthorized Response schema:</p></blockquote>
-
-```schema
-[[equipment:model:response_getEquipment_401]]
+``` http
+GET /equipment HTTP/1.1
+Authorization: <A Invalid Token>
+Host: fuse-equipment-api-sandbox.herokuapp.com
 ```
```
Comment (jeffhsta) [inline source/includes/equipment_api/_equipment.md]: There is a double blank line here

**Comment by lucaslago** [comment]
👍

**Comment by jeffhsta** [comment]
:+1:

**Pull request merged**